pennylane/measurements.py

```diff
@@ -690,7 +690,9 @@ def counts(op=None, wires=None):
     Args:
         op (Observable or None): a quantum observable object
         wires (Sequence[int] or int or None): the wires we wish to sample from, ONLY set wires if
-        op is None
+            op is None
+        all_outcomes(bool): determines whether the returned dict will contain only the observed
+            states (default), or whether it will display all possible outcomes for the system
```

Comment:
Note: when an observable is specified we have eigenvalues as keys.
```suggestion
            outcomes (default), or whether it will display all possible outcomes for the system
```